evaluator/notifications.go

```diff
@@ -89,14 +89,14 @@ func publishNewAdvisoriesNotification(tx *gorm.DB, inventoryID, accountName stri
 	if err != nil {
 		return errors.Wrap(err, "writing message to notifications publisher failed")
 	}
+	utils.Log("account name", accountName, "inventoryID", inventoryID, "notification", notification).
+		Info("notification sent successfully")
```

Comment:
please change the log level to Debug, we don't need it i.e. in prod and we also probably should not log which unpatched errata each customer has. This could be considered as sensitive information.